solutions/src/main/java/ru/sberbank/school/task02/service/Main.java

```diff
@@ -0,0 +1,26 @@
+package ru.sberbank.school.task02.service;
+
+
+import ru.sberbank.school.task02.util.Beneficiary;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.ExternalQuotesServiceDemo;
+import ru.sberbank.school.task02.util.Symbol;
+
+import java.math.BigDecimal;
+
+public class Main {
+    public static void main(String[] args) {
+
+        ServiceFactoryImpl serviceFactory = new ServiceFactoryImpl();
+
+        System.out.println(
+                serviceFactory
+                        .getExtendedFxConversionService(new ExternalQuotesServiceImpl())
+                        .convertReversed(ClientOperation.SELL, Symbol.USD_RUB, BigDecimal.valueOf(77700), Beneficiary.CLIENT));
```

Comment:
Line is longer than 120 characters (found 127).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1761994309>*